OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 11

# Description
Lint fix for Reports component, this is phase 11.

## Files changed
"All .js and .jsx files in src/components/Reports/ProjectMemberTable."

eslint test:
![CleanShot 2023-10-05 at 00 50 40@2x](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/b91cbd85-313e-46a2-8227-dc0da2bdaee1)



# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/ProjectMemberTable/**/*.{js,jsx}`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!
